### PR TITLE
Pug ship descriptions

### PR DESCRIPTION
**Pull request by McloughlinGuy**
## Summary
I wrote descriptions for the pug warships that invade human space. I know that you can't actually see them in game, but the Archon and Arfecta both have descriptions (Archon getting a description was the subject of my first ever github issue) and you won't be able to even obtain the Archon. Plus I'm pretty sure the Ka'het have descriptions even though they can't be captured. I made an issue for this (#5284) a while ago, which I closed. However I decided I wanted to write descriptions for them now. (No I don't think the pug will ever let you buy them.) Anyways, it took a lot of refining and rewriting parts to get it into a state I was happy with, and if you have any suggestions or feedback I'm definitely open to it. This is the first time I've ever done a PR where I actually am contributing my writing (Chaser one doesn't count) so I hope it meets the quality standards.

### CODE REVIEW

**Comment by Hurleveur** [comment]
I like it but I think we could get a bit less repetitive in the description? Like all ships have high shield and hull regen (that part not mentioned?) and low hps we got it

**Comment by McloughlinGuy** [comment]
> I like it but I think we could get a bit less repetitive in the description? Like all ships have high shield and hull regen (that part not mentioned?) and low hps we got it. Maybe add a bit more to it? Idk, no writer myself edit: the tier 1 repetition feels a bit heavy too mb we can just say that one in different ways though The high innate energy generation also comes to mind.

Yeah, understandable. Might try to modify the descriptions to give less repetition.

**Comment by McloughlinGuy** [comment]
> I like it but I think we could get a bit less repetitive in the description? Like all ships have high shield and hull regen (that part not mentioned?) and low hps we got it. Maybe add a bit more to it? Idk, no writer myself edit: the tier 1 repetition feels a bit heavy too mb we can just say that one in different ways though The high innate energy generation also comes to mind.

As per your suggestion, I shortened the descriptions. Each description still says "this is the smallest/medium/largest" like the original, but I made sure each description mentions something different, like for the small warship its lack of gun ports but fast speed, for the medium (as per your suggestion) integrated energy systems, and the last one I just kept the same because now its the only one which is mentioning the fast shield regeneration but low shields and hull. (Also I tested it, they don't have hull regeneration.)

**Review by Hurleveur — COMMENTED** [review]
Looks much better now! I must have thought of the arfecta that has hull regen.
Btw I don't know what you're doing with the white lines but I think the convention is 3 between each ship.

**Review by Hurleveur — APPROVED** [review]
I am no content reviewer but as smbd that sometimes goes around in the files I quite like this change.

File: data/pug/pug.txt [hunk 1]
```diff
@@ -666,7 +666,7 @@ ship "Pug Maboro"
 	explode "pug large explosion" 20
 	explode "pug huge explosion" 5
 	"final explode" "final explosion large"
-	description "This is the largest Pug warship."
+	description `The Maboro appears to be the Pug's capital warship. Its shield matrix and hull are much weaker than human capital ships, however it is able to regenrate its shields quickly. Its powerful weapory and advanced anti-missile systems make it a ship that only the strongest human capital ships are able to oppose.`
```
Comment (roadrunner56) [inline data/pug/pug.txt:669]:
```suggestion
	description `The Maboro appears to be the Pug's capital warship. Its shield matrix and hull are much weaker than human capital ships; however, it is able to regenerate its shields quickly. Its powerful weaponry and advanced anti-missile systems make it a ship that only the strongest human capital ships are able to oppose.`
```

File: data/pug/pug.txt [hunk 2]
```diff
@@ -609,7 +609,7 @@ ship "Pug Enfolta"
 	explode "pug medium explosion" 25
 	explode "pug large explosion" 15
 	"final explode" "final explosion medium"
-	description "This is a medium-sized Pug warship."
+	description `The Enfolta is a medium-sized Pug warship. It has a powerful arnament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on a fleet of smaller human ships.`
```
Comment (roadrunner56) [inline data/pug/pug.txt:612]:
```suggestion
	description `The Enfolta is a medium-sized Pug warship. It has a potent armament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on a fleet of smaller human ships.`
```

File: data/pug/pug.txt [hunk 3]
```diff
@@ -554,7 +554,7 @@ ship "Pug Zibruka"
 	explode "pug medium explosion" 20
 	explode "pug large explosion" 10
 	"final explode" "final explosion small"
-	description "The Zibruka is the smallest Pug ship that you know of."
+	description `The Zibruka is the smallest Pug warship. Despite having only two gun ports, its advanced guns combined with fast and efficient engine systems make it a fearsome ship to face off against, especially in swarms.`
```
Comment (roadrunner56) [inline data/pug/pug.txt:557]:
```suggestion
	description `The Zibruka is the smallest Pug warship that you know of. Despite having only two gun ports, its advanced guns combined with fast and efficient engine systems make it a fearsome ship to face off against, especially in swarms.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/pug/pug.txt (hunk 2 above)
Comment (McloughlinGuy) [reply data/pug/pug.txt:612]: I think there might be a better word than potent or powerful

Comment (McloughlinGuy) [reply data/pug/pug.txt:612]: @roadrunner56 do you think "formidable" would be a good word?

Comment (roadrunner56) [reply data/pug/pug.txt:612]: Yeah, that would be better.

Comment (McloughlinGuy) [reply data/pug/pug.txt:612]: > 

Alright, I will change it to that.

**Review by roadrunner56 — APPROVED** [review]

File: data/pug/pug.txt [hunk 4]
```diff
@@ -609,7 +609,7 @@ ship "Pug Enfolta"
 	explode "pug medium explosion" 25
 	explode "pug large explosion" 15
 	"final explode" "final explosion medium"
-	description "This is a medium-sized Pug warship."
+	description `The Enfolta is a medium-sized Pug warship. It has a formidable arnament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on a fleet of smaller human ships.`
```
Comment (Hatrask) [inline data/pug/pug.txt:612]:
```suggestion
	description `The Enfolta is a medium-sized Pug warship. It has a formidable ornament of advanced weaponry supported by powerful integrated energy generation. A single Enfolta can take on an entire fleet of smaller human ships.`
```

Comment (McloughlinGuy) [reply data/pug/pug.txt:612]: I'll change it to "an entire" and arnament to armament

**Comment by McloughlinGuy** [comment]
(I'm unsure about saying weapons instead of guns in the Zibruka description, I might change it back later, especially since every description says weapons or weaponry now.)

**Comment by Ferociousfeind** [comment]
I agree with these expanded descriptions. Though, I'm not too sure how much more powerful the Enfolta is than the average human war fleet. Is it really capable of taking on an entire fleet of light warships and interceptors...?

**Comment by McloughlinGuy** [comment]
> I agree with these expanded descriptions. Though, I'm not too sure how much more powerful the Enfolta is than the average human war fleet. Is it really capable of taking on an entire fleet of light warships and interceptors...?

I might change it to "multiple" rather than "an entire" or maybe just say "interceptors" rather than "smaller human ships"
Thanks for the suggestion!

**Pull request merged**